docs/contributing/gatsby-governance-model.md

```diff
@@ -6,63 +6,262 @@ title: Gatsby's Governance Model
+This role does not require any extra responsibilities or time commitment. We hope you stick around and keep participating!
+
+If you're interested in reaching the next level and becoming a Maintainer, you can begin to explore some of those responsibilities in the next section.
+
+#### Nomination Process
+
+n/a
+
+### Level 2 (L2) - Maintainer
+
+The **Maintainer** role is available to any contributor who wants to join the team and take part in the long-term maintenance of Gatsby.
+
+The Maintainer role is critical to the long-term health of Gatsby. Maintainers support the **Team** members and together they act as the first line of defense when it comes to new issues, pull requests and Discord activity. Maintainers are most likely the first people that a user will interact with on Discord or GitHub.
+
+**A Maintainer is not required to write code!** Some Maintainers spend most of their time inside of Discord, maintaining a healthy community there. Maintainers can also be thought of as **Moderators** on Discord and carry special privileges for moderation.
+
+#### Recognized contributions
+
+There is no strict minimum number of contributions needed to reach this level, as long as you can show **sustained** involvement over some amount of time (at least a couple of weeks).
+
+- **GitHub:** Submitting non-trivial pull requests and RFCs
+- **GitHub:** Reviewing non-trivial pull requests and RFCs
+- **Discord:** Supporting users in Discord, especially in the "Help" category
+- **Discord:** Active participation in RFC calls and other events
+- **GitHub + Discord:** Triaging and confirming user issues
+- This list is incomplete! Similar contributions are also recognized.
+
+#### Privileges
+
+- All privileges of the [Contributor role](#TODO), plus...
+- Invitation to the `@Maintainer` role on [Discord](https://gatsby.dev/discord)
+- Invitation to the `Maintainers` team on GitHub.
+- New name color on Discord: **Blue**.
+- Invitation to the private `#maintainers` channel on Discord.
+- Ability to moderate Discord to remove spam, harmful speech, etc.
+- Ability to join the `@Moderator` role on Discord (optional, opt-in).
+- Ability to review GitHub PRs.
+- Ability to vote on _some_ initiatives (see [Voting](#voting) below).
+
+#### Responsibilities
+
+- Participate in the project as a team player.
+- Bring a friendly, welcoming voice to the Gatsby community.
+- Be active on Discord, especially in the "Help" category.
+- Triage new issues.
+- Review pull requests.
+- Merge some, non-trivial community pull requests.
+- Merge your own pull requests (once reviewed and approved).
+
+#### Nomination
+
+To be nominated, a nominee is expected to already be performing some of the responsibilities of a Maintainer over the course of at least a couple of weeks.
+
+In some rare cases, this role may be revoked by an **Admin**. However, under normal circumstances this role is granted for as long as the contributor wishes to engage with the project.
+
+### Team
```

Comment:
Team is referenced as L3 on line 180, if it is L3 it would be good to have that in the heading. Since it didn't look like the other level headings I got confused for a bit and didn't know we were talking about a new level